cmd/model/model.go

```diff
@@ -16,9 +16,6 @@ limitations under the License.
 package model
 
 import (
```

Comment:
want to confirm the change in this file is intended

Reply:
Yeah, the store id should not be required on the models because `model validate` does not need it